docs(cli): explain optional devContainer argument for okteto up (#1303)
okteto/docs: src/content/reference/okteto-cli.mdx

$ okteto up [devContainer]
```

> The `devContainer` argument is the name of a Development Container defined in the [`dev`](reference/okteto-manifest.mdx#dev-object-optional) section of your Okteto Manifest, and it is optional. When you specify it, Okteto activates that Development Container directly, without scanning the available Development Containers or asking you to pick one from a list. If you omit it and your manifest defines more than one Development Container, Okteto prompts you to choose.

Reviewer: ```suggestion
The `devContainer` argument is optional and it's the name of a Development Container defined in the [`dev`](reference/okteto-manifest.mdx#dev-object-optional) section of your Okteto Manifest. If you omit it, `okteto up` activates the only Development Container in your Okteto Manifest, or presents a selector when there's more than one.
```
